common/static/coffee/src/discussion/discussion_module_view.coffee

```diff
@@ -10,10 +10,11 @@ if Backbone?
       "click .discussion-paginator a": "navigateToPage"
 
     page_re: /\?discussion_page=(\d+)/
-    initialize: ->
+    initialize: (options) ->
       @toggleDiscussionBtn = @$(".discussion-show")
       # Set the page if it was set in the URL. This is used to allow deep linking to pages
       match = @page_re.exec(window.location.href)
+      @context = options.context or "course"  # allowed values are "course" or "standalone"
```

Comment:
As I mentioned on the specific commit, I'd like to talk about us doing this as an editing option instead of linking it to context (since we don't pervasively need the context in the JavaScript code).